test/models/vote_test.rb

```diff
@@ -0,0 +1,37 @@
+require "test_helper"
+
+describe Vote do
+  describe "relations" do
+    before do
+      @vote = votes(:one)
+    end
+    it "is valid when a vote connects a user" do
+      user = User.first
+      user.votes << @vote
+
+      @vote.user_id.must_equal user.id
+    end
+
+    it "is valid when a vote connects a work" do
+      work = Work.first
+      work.votes << @vote
```

Comment:
I think that the "is valid" in the title of these tests is a little confusing, since they don't test validations.